### PR TITLE
Settings page 

### PR DESCRIPTION
**Pull request by Athenstan**
Need to check if it is working with Alans backend branch but it seems to be working well with the S3 for the profile photos.

### CODE REVIEW

File: node_modules/.yarn-integrity
Comment (Harin329) [inline node_modules/.yarn-integrity:1]: can we remove this please :)

File: yarn.lock
Comment (Harin329) [inline yarn.lock:1]: this as well

**Comment by Harin329** [comment]
Some screenshots of key features would be nice!

**Comment by Athenstan** [comment]
Screenshot 2023-05-18 at 2 38 52 PM
New Settings page ^ 

Screenshot 2023-05-18 at 2 39 05 PM
Clicking on Edit will bring up the edit tab ^ 

Screenshot 2023-05-18 at 2 39 17 PM
Clicking on the profile photo will allow you to change the profile picture ^ 


Screenshot 2023-05-18 at 2 39 28 PM
Profile Picture is now saved and displayed on the user settings page ^ 

Note that the other branch from Allan should be merged it for the backend portion of this

**Review by Harin329 — APPROVED** [review]

**Comment by Harin329** [comment]
Thanks for including screenshots!

**Pull request merged**